nabboot/nabboot.py

```diff
@@ -52,8 +54,15 @@ def set_system_led(shutdown):
     with open("/sys/class/leds/led0/trigger", "w") as f:
         f.write("none")
 
-    with open("/sys/class/leds/led0/brightness", "w") as f:
-        f.write("0")
+    kernel_version = os.popen("uname -r").read()
```

Comment:
Could we use `platform.release()` instead?

Reply:
> Could we use `platform.release()` instead?

As you wish. I do hope it will keep returning the same content as `uname -r `in the future...